src/common/exif.cc

```diff
@@ -692,6 +692,21 @@ static bool _exif_decode_iptc_data(dt_image_t *img, Exiv2::IptcData &iptcData)
   }
 }
 
+static bool dt_exif_read_exif_tag(Exiv2::ExifData &exifData, Exiv2::ExifData::const_iterator *pos, string key)
```

Comment:
`dt_exif_read_exif_tag` => `_exif_read_exif_tag`

Prefix `dt_` is only used for exported routines.

Reply:
This functions already existed, they were just moved up. Should I change them anyway?

Reply:
Up to you, not mandatory for approving this PR. It would be nice, I'm trying to always to clean-up when working even if not on my part of code... And yes I know it is painful! My goal is to have all the code looking the same at some point to be sure every dev is comfortable in the whole code base.